### PR TITLE
PCHR-1223: Styling contact summary tabs

### PR DESCRIPTION
**Pull request by brunoqueiros**
This PR override the contact summary tabs styles to looks like SSP. All changes were made on contact.css file over `.page-civicrm-contact` namespace. It was created a map on `_tabs.scss` to define each icon design.

***Before***
pchr-1223-before

***After***
pchr-1223-after

### CODE REVIEW

**Comment by omarabuhussein** [comment]
some tabs are not covered , can you please visit this page http://civihr-dev.civihrhosting.co.uk/index.php?q=civicrm/contact/view&reset=1&cid=158 and check which tabs are not covered in this PR

File: hrui/js/contact.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      'Bank Details': 'fa fa-university',
+      'Identification': 'fa fa-credit-card',
+      'Immigration': 'fa fa-plane',
+      'Career History': 'fa fa-history',
+      'Medical & Disability': 'fa fa-wheelchair',
+      'Qualifications': 'fa fa-certificate',
+    };
+
+    for (var iconTabName in iconsMap) {
+      $('.crm-contact-tabs-list li a[title="' + iconTabName + '"]').prepend('<span class="' + iconsMap[iconTabName] + '" />');
```
Comment (danilo-compucorp) [inline hrui/js/contact.js]: Do we really need Javascript for this? Couldn't this be done using CSS only?
E.g.:
```
.crm-contact-tabs-list li a[title="Job Contract"]::before {
  font-family: 'FontAwesome';
  content: '\f016';
}
```

**Comment by danilo-compucorp** [comment]
Just need to update the description (regarding the icons with js). Then it's approved

**Comment by omarabuhussein** [comment]
looks good . ... one note though ... is there is a way in CSS to set a default icon for any tab that is not part of the defined ones .

**Comment by brunoqueiros** [comment]
@omarabuhussein  yes there is, just need to check which one will be the default icon.

**Comment by omarabuhussein** [comment]
@brunoqueiros this one http://fontawesome.io/icon/puzzle-piece/

**Comment by omarabuhussein** [comment]
from me side it's approved , if @danilo-compucorp just can confirm the latest modification you are good to go. :+1:

**Comment by danilo-compucorp** [comment]
It's approved

**Pull request merged**